inrupt/pod-browser
# Remove helpdesk link

[ticket](https://inrupt.atlassian.net/browse/APPS-1202)

## Description
This PR removes the link to the Jira helpdesk

## Changes
- Removed the help link
- Updated the test snapshot

## Testing
- Visit the login page
- verify you do not see the help link

## Commit checklist


## Interested parties
@KyraAssaad 

## Screenshots/captures

### Before
![before](https://user-images.githubusercontent.com/35955/157605702-7bc72d50-07a7-467d-bcef-e4eb11ed1be0.png)

### After
![after](https://user-images.githubusercontent.com/35955/157600760-f8a99a88-98f0-4cce-ae75-f1691cd50385.png)
